pallets/nfts/src/tests.rs

```diff
@@ -2206,6 +2263,12 @@ fn cancel_approval_works() {
 		);
 
 		assert_ok!(Nfts::cancel_approval(RuntimeOrigin::signed(account(2)), 0, 42, account(3)));
+		assert!(events().contains(&Event::<Test>::ApprovalCancelled {
```

Comment:
Again, this should not be tested here. Is this event emitted in `check_approval`?

Reply:
This is inside cancel_approval